ledger/controllers/grants.js

```diff
@@ -303,6 +341,31 @@ v3.read = {
   }
 }
 
+/*
+  GET /v4/grants
+*/
+
+v4.read = {
+  handler: getGrantV4(4, chooseFromAvailablePromotionsByType),
```

Comment:
yeah, with so many different promotions often being added at different times, this is useful to isolate different parts and scope them / help isolate issues

Reply:
I was thinking that the handler function would already be isolated because of the name.  do we expect to have v5 routes to use the `getGrantV4` handler?

Reply:
Oh, you're saying the name of the function that creates the closure should change. yeah. i can modify that